ramls/edge-oai-pmh.raml

```diff
@@ -11,14 +11,23 @@ types:
 traits:
   resps: !include traits/resps.raml
   params: !include traits/params.raml
+  apiKeyParam: !include traits/apiKeyParam.raml
 /oai:
   displayName: OAI-PMH
   get:
     description: 'Run OAI-PMH request'
-    is: [params, resps]
+    is: [params, apiKeyParam, resps]
   post:
     description: 'Run OAI-PMH request'
-    is: [params, resps]
+    is: [params, apiKeyParam, resps]
+  /{apiKeyPath}:
+    displayName: OAI-PMH
+    get:
+      description: 'Run OAI-PMH request'
+      is: [params, resps]
+    post:
+      description: 'Run OAI-PMH request'
+      is: [params, resps]
```

Comment:
I just realized looking at this that we need to update the RAML and module descriptor too... Do you guys mind taking it from here?